integration_tests/src/main/python/cast_test.py

```diff
@@ -107,7 +107,7 @@ def test_cast_string_date_invalid_ansi_databricks(invalid):
         error_message="DateTimeException")
 
 # test databricks, ANSI mode, valid values
-@pytest.mark.skipif(not is_databricks91_or_later(), reason="Spark versions(< 320) not support Ansi mode when casting string to date")
+@pytest.mark.skipif(not is_databricks104_or_later(), reason="Spark versions(< 320) not support Ansi mode when casting string to date")
```

Comment:
Shouldn't this test apply to everything >= Spark 3.2 and not just Databricks?  I didn't see an equivalent test for non-Databricks with Spark 3.2+, seems to be a dropped test case for those platforms.

Reply:
You are right. Updated the test to run for Spark-3.2+ . PTAL.